content/the-ember-times-issue-178.md

```diff
@@ -26,15 +28,15 @@ tags:
 
 ---
 
-## [2. Section title in sentence case 🐹](section-url)
+## Addon roundup 🙌
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+We wanted to highlight some recent happenings around the web! Check them out and give them a try! 😎
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+<!-- alex ignore easy -->
+* Do you want to use OAuth in your Ember app? [Sinan Keskin (@sinankeskin)](https://github.com/sinankeskin) wrote a new addon, [ember-oauther](https://github.com/sinankeskin/ember-oauther), which allows you to sign in and register a user with OAuth easily! OAuth is an open standard for **access delegation**, commonly used as a way to register or log into a web app via their Facebook, Google, Microsoft, etc. login.
```

Comment:
optional:

```suggestion
* Do you want to use OAuth in your Ember app? [Sinan Keskin (@sinankeskin)](https://github.com/sinankeskin) wrote a new addon, [ember-oauther](https://github.com/sinankeskin/ember-oauther), which allows you to sign in and register a user with OAuth easily! OAuth is an open standard for **access delegation**, commonly used to help a user register or log in to a web app via their account for Facebook, Google, Microsoft, etc.
```

Reply:
brought this over to the main branch!, feel free to commit edits anytime!